test/test_download.py

```diff
@@ -64,6 +64,8 @@ def _file_md5(fn):
 
 
 class TestDownload(unittest.TestCase):
+    _multiprocess_shared_ = True
```

Comment:
I think there should be a link to https://nose.readthedocs.org/en/latest/doc_tests/test_multiprocess/multiprocess.html, to make it clear that this is not from `unittest.TestCase` (that was where I first looked for its documentation).